packages/react/package.json

```diff
@@ -62,7 +62,7 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^8.0.2",
-    "@slashid/slashid": "3.15.0",
+    "@slashid/slashid": "file:.yalc/@slashid/slashid",
```

Comment:
How does it work? The version is referenced in a gitignored file? :thinking:

Reply:
This is just a placeholder, but it's a tool for publishing packages on local machine for development

I'll change this to the actual version once its published

https://github.com/wclr/yalc

Reply:
done